src/main/java/com/org/jenkins/custom/jenkins/distribution/service/util/Util.java

```diff
@@ -42,4 +47,21 @@ public void cleanupTempDirectory(File file) {
     public JSONObject convertPayloadToJSON(String payload) throws Exception{
         return new JSONObject(payload);
     }
+
+    public ResponseEntity<Resource> returnResource(HttpHeaders headers, File file, InputStreamResource resource){
```

Comment:
This class has no internal state, you could make this method static and simply call `Util.returnResource(...)`.

Reply:
Sounds Perfect :+1:

Reply:
:+1: